bayes_opt/logger.py

```diff
@@ -62,11 +33,21 @@ class ScreenLogger(_Tracker):
     _colour_regular_message = Fore.RESET
     _colour_reset = Fore.RESET
 
-    def __init__(self, verbose: int = 2, is_constrained: bool = False) -> None:
+    def __init__(
+        self,
+        verbose: int = 2,
+        is_constrained: bool = False,
+        params_config: Mapping[str, ParameterConfig] | None = None,
+    ) -> None:
         self._verbose = verbose
         self._is_constrained = is_constrained
+        self._params_config = params_config
```

Comment:
It's not ideal that we need to copy this object over whole, but maybe there is no other way (?)